editor/src/clj/editor/pipeline/texture_set_gen.clj

```diff
@@ -38,14 +38,14 @@
 
 (defn- Rect->map
   [^TextureSetLayout$Rect rect]
-  {:path (.id rect)
-   :index (.index rect)
-   :x (.x rect)
-   :y (.y rect)
-   :page (.page rect)
-   :width (.width rect)
-   :height (.height rect)
-   :rotated (.rotated rect)})
+  {:path (.getId rect)
+   :index (.getIndex rect)
+   :x (.getX rect)
+   :y (.getY rect)
+   :page (.getPage rect)
+   :width (.getWidth rect)
+   :height (.getHeight rect)
+   :rotated (.getRotated rect)})
```

Comment:
Part of adding these structs to the public api, i.e. using the accessors instead.